pypeit/fluxcalibrate.py

```diff
@@ -138,15 +140,17 @@ def flux_calib(self, sobjs, wave, zeropoint, meta_table):
                wavelength array for sensitivity function (nspec,)
             zeropoint (ndarray):
                sensitivity function
-            meta_table (table):
-               astropy table containing meta data for sensitivity function
+            algorithm (str):
+                SensFunc algorithm
+            sens_table (table.Table):
```

Comment:
`sens` or `sens_table`?

Reply:
Good catch!  Fixed.  I used `sens`, but I can change it to `sens_table`, if that's preferred.

Reply:
I'm fine with either.